views/beneficial-owner-other.html

```diff
@@ -66,6 +66,8 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
 
         {% set sanctionsListText = "Is it on the sanctions list?" %}
         {% set sanctionsListHintText = "This means that it is subject to sanctions under the Sanctions and Anti-Money Laundering Act 2018." %}
+        {% set sanctionsListNoText = "No, they are not on the Sanctions List" %}
```

Comment:
I think the `sanctionsListNoText`  should be `No, it is not on the Sanctions List`

Reply:
This is not what is on the screenshot but it should be "it". Will need to put a note on the ticket for the testers about this.